### PR TITLE
drivers: add buck status power get command to shell

### PR DESCRIPTION
**Pull request by magp-nordic**
JIRA: NRFX-3849

### CODE REVIEW

File: drivers/npmx/npmx_shell.c [hunk 1]
```diff
@@ -962,7 +962,6 @@ static int cmd_buck_mode_set(const struct shell *shell, size_t argc, char **argv
 
 	if (argc < 3) {
 		shell_error(shell, "Error: missing buck mode.");
-		return 0;
```
Comment (wilu-nordic) [inline drivers/npmx/npmx_shell.c:965]: Why it is removed?

File: drivers/npmx/npmx_shell.c [hunk 2]
```diff
@@ -1670,13 +1709,24 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_buck_gpio,
 					 "Select GPIO used as buck's PWM forcing", NULL),
 			       SHELL_SUBCMD_SET_END);
 
+/* Creating subcommands (level 4 command) array for command "buck status power". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_buck_status_power,
+			       SHELL_CMD(get, NULL, "Get BUCK status", cmd_buck_status_power_get),
```
Comment (wilu-nordic) [inline drivers/npmx/npmx_shell.c:1714]:
```suggestion
			       SHELL_CMD(get, NULL, "Get BUCK power status", cmd_buck_status_power_get),
```

File: drivers/npmx/npmx_shell.c (hunk 1 above)
Comment (magp-nordic) [reply drivers/npmx/npmx_shell.c]: Accidentally removed while rebasing, fixed.

**Review by wilu-nordic — APPROVED** [review]

File: drivers/npmx/npmx_shell.c [hunk 3]
```diff
@@ -1008,6 +1008,47 @@ static int cmd_buck_mode_set(const struct shell *shell, size_t argc, char **argv
...
+		return 0;
+	}
+
+	if (argc < 2) {
+		shell_error(shell, "Error: missing buck instance index.");
+		return 0;
+	}
+
+	int err = 0;
+	uint8_t buck_indx = (uint8_t)shell_strtoul(argv[1], 0, &err);
```
Comment (masz-nordic) [inline drivers/npmx/npmx_shell.c:1026]: Can you use `CLAMP` here? And also in line 969, I think it was skipped there.

**Review by masz-nordic — COMMENTED** [review]
Please also change commit title to:
`drivers: npmx: shell: add buck status power get command`

**Review by masz-nordic — APPROVED** [review]

**Pull request merged**